kits/rosie/rosie_demo.cpp

```diff
@@ -339,9 +380,9 @@ int main(){
       }
       //
       //            % Joystick Input for Omnibase Control        
-      double x_vel = base.max_lin_speed_ * (latest_phone_fbk[0].io().a().hasInt(8) ? latest_phone_fbk[0].io().a().getInt(8) : latest_phone_fbk[0].io().a().getFloat(8)); 
-      double y_vel = -1*base.max_lin_speed_ * (latest_phone_fbk[0].io().a().hasInt(7) ? latest_phone_fbk[0].io().a().getInt(7) : latest_phone_fbk[0].io().a().getFloat(7));
-      double rot_vel = base.max_rot_speed_ * (latest_phone_fbk[0].io().a().hasInt(1) ? latest_phone_fbk[0].io().a().getInt(1) : latest_phone_fbk[0].io().a().getFloat(1)); 
+      double x_vel = base_max_lin_speed * (latest_phone_fbk[0].io().a().hasInt(8) ? latest_phone_fbk[0].io().a().getInt(8) : latest_phone_fbk[0].io().a().getFloat(8)); 
```

Comment:
Few things here:

1) You're accessing the same fields here, so just do something like `auto& bankA = latest_phone_fbk[0].io().a();`
2) It's a bit nicer if you just have a function which corresponds to retrieving this value for you, which can also be appropriately named based on its intention. Some simple example would be:
```c++
double x_vel = base_max_lin_speed * getUserVelocityInput();
```

However... It gets messy since you have to retrieve the feedback through `latest_phone_fbk`. That indicates that some sort of structure should be created, in which the `latest_phone_fbk` is encapsulated. So this sounds out of the scope of this PR, but relevant for future PRs

Reply:
Just realizing that our about-to-be-released C++ API has a "mobile_io" class which would pretty much take care of this...